core/src/androidTest/java/in/testpress/core/database/CommentDaoTest.kt

```diff
@@ -0,0 +1,43 @@
+package `in`.testpress.core.database
+
+import `in`.testpress.database.entities.CommentEntity
+import `in`.testpress.database.entities.ContentObject
+import `in`.testpress.database.entities.ProfileDetails
+import `in`.testpress.util.getOrAwaitValue
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class CommentDaoTest: DbTestMixin() {
+
+    @Test
+    fun readShouldReturnInsertedData() {
+        insertCommentIntoDb()
+
+        val fetchedComment = db.commentDao().getAll().getOrAwaitValue()
+        Assert.assertEquals(createCommentFixture(),fetchedComment)
+    }
+
+    private fun insertCommentIntoDb() {
+        createCommentFixture().forEach {
```

Comment:
You are not creating anything here, it is an object. This can be renamed as `getComments()`